### PR TITLE
[mask-detection] Added PyTorch to the mask-detection demo

### PR DESCRIPTION
**Pull request by guy1992l**
Added the PyTorch framework to the mask detection demo. Now supports TF.Keras and PyTorch.

### CODE REVIEW

**Comment by aviaIguazio** [comment]
@guy1992l please move your commit to 0.9.x-dev
